pkg/server/ca/ca_test.go

```diff
@@ -179,20 +179,20 @@ func (s *CATestSuite) TestSignX509SVIDWithSubject() {
 	}{
 		{
 			name:     "empty subject",
-			expected: "O=SPIRE,C=US",
+			expected: "OU=TestUnit,O=TestOrg,2.5.4.45=#13203933323965323863393434383738376466306663623363363535363035653531",
```

Comment:
can you add a test case where CA has a Country? so we can verify it is propagated?

Reply:
No longer relevant since we've removed CA subject inheritance.